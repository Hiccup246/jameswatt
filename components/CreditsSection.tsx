import BrittanyChiangIcon from "./Icons/BrittanyChiangIcon";
import FontAwesomeIcon from "./Icons/FontAwesomeIcon";
import Icons8Icon from "./Icons/Icons8Icon";
import SectionLayout from "./SectionLayout";
import Image from "next/image";
import FreshIcon from "./Icons/FreshIcon";

export default function CreditsSection() {
    return (
        <SectionLayout isBgColorPrimary={false}>
            <h1 className="text-center text-2xl sm:text-3xl font-bold mb-2">
                The Work Which Inspired This Site
            </h1>
            
            <h2 className="text-center mb-10">
                &quot;Good artists borrow, great artists steal&quot; - Pablo Picasso
            </h2>

            <div className="text-center">
                <ul>
                    <li className="group hover mb-5">
                        <a className="group-hover:underline pr-2" href="https://icons8.com/" aria-label="Icons8 Homepage">
                            Icons8 (Icons)
                        </a>

                        <a className="inline-block h-4 w-4 opacity-0 -rotate-45 group-hover:opacity-70 duration-500 transition-opacity-transform ease-in-out translate-x-[-8px] group-hover:transform-none align-middle mb-1" href="https://icons8.com/" aria-label="Icons8 Homepage">
                            <Icons8Icon />
                        </a>
                    </li>

                    <li className="group hover mb-5">
                        <a className="group-hover:underline pr-2" href="https://fontawesome.com/" aria-label="Font Awesome Homepage">
                            Font Awesome (Icons)
                        </a>

                        <a className="inline-block h-4 w-4 opacity-0 -rotate-45 group-hover:opacity-70 duration-500 transition-opacity-transform ease-in-out translate-x-[-8px] group-hover:transform-none align-middle mb-1" href="https://fontawesome.com/" aria-label="Font Awesome Homepage">
                            <FontAwesomeIcon />
                        </a>
                    </li>

                    <li className="group hover mb-5">
                        <a className="group-hover:underline pr-2" href="https://brittanychiang.com/" aria-label="Brittany Chiang Website">
                            Brittany Chiang (Inspired the programming experience component)
                        </a>

                        <a className="inline-block h-4 w-4 opacity-0 -rotate-45 group-hover:opacity-70 duration-500 transition-opacity-transform ease-in-out translate-x-[-8px] group-hover:transform-none align-middle mb-2" href="https://brittanychiang.com/" aria-label="Brittany Chiang Website">
                            <BrittanyChiangIcon />
                        </a>
                    </li>

                    <li className="group hover mb-5">
                        <a className="group-hover:underline pr-2" href="https://marguerite.io/" aria-label="Marguerite Roth Website">
                            Marguerite Roth (Inspired the bookshelf component)
                        </a>

                        <a className="h-6 w-6 relative inline-block opacity-0 -rotate-45 group-hover:opacity-70 duration-500 transition-opacity-transform ease-in-out translate-x-[-8px] group-hover:transform-none align-middle mb-1" href="https://marguerite.io/" aria-label="Marguerite Roth Website">
                            <Image src="/marguerite.webp"
                                   fill
                                   sizes="(min-width: 640px) 300px,
                                          (min-width: 785px) 300px,
                                          300px"
                                   alt="Marguerite Roth Icon" />
                        </a>
                    </li>

                    <li className="group hover mb-5">
                        <a className="group-hover:underline pr-2" href="https://fresh.deno.dev/" aria-label="Fresh Framework Homepage">
                            Fresh Framework (Dripping Design + Project Showcase)
                        </a>

                        <a className="inline-block h-6 w-6 opacity-0 -rotate-45 group-hover:opacity-70 duration-500 transition-opacity-transform ease-in-out translate-x-[-8px] group-hover:transform-none align-middle mb-1" href="https://fresh.deno.dev/" aria-label="Fresh Framework Homepage">
                            <FreshIcon />
                        </a>
                    </li>

                    <li className="group hover mb-5">
                        <a className="group-hover:underline pr-2" href="https://www.joshwcomeau.com/" aria-label="Josh Comeau Website">
                            Josh Comeau (Moon Icon)
                        </a>

                        <a className="relative inline-block h-6 w-6 opacity-0 -rotate-45 group-hover:opacity-70 duration-500 transition-opacity-transform ease-in-out translate-x-[-8px] group-hover:transform-none align-middle mb-1" href="https://www.joshwcomeau.com/" aria-label="Josh Comeau Website">
                            <Image src="/josh-comeau.webp"
                                   fill
                                   sizes="(min-width: 640px) 170px,
                                          (min-width: 785px) 170px,
                                          170px"
                                   alt="Josh Comeau Icon" />
                        </a>
                    </li>
                </ul>
            </div>
        </SectionLayout>
    )
}
