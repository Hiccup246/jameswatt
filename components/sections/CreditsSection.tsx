import BrittanyChiangIcon from "../Icons/BrittanyChiangIcon";
import FontAwesomeIcon from "../Icons/FontAwesomeIcon";
import Icons8Icon from "../Icons/Icons8Icon";
import SectionLayout from "../layouts/SectionLayout";
import Image from "next/image";
import FreshIcon from "../Icons/FreshIcon";
import NuxtIcon from "../Icons/NuxtIcon";
import Marguerite from "../../public/marguerite.webp";
import JoshComeau from "../../public/josh-comeau.webp";

export default function CreditsSection() {
  return (
    <SectionLayout isBgColorPrimary={false}>
      <h1 className="text-center text-2xl font-bold sm:text-3xl">
        The Work Which Inspired This Site
      </h1>

      <h2 className="mb-16 text-center">
        &quot;Good artists borrow, great artists steal&quot; - Pablo Picasso
      </h2>

      <div className="text-center">
        <ul>
          <li className="hover group mb-5">
            <a
              className="pr-2 group-hover:underline"
              href="https://icons8.com/"
              aria-label="Icons8 Homepage"
            >
              Icons8 (Icons)
            </a>

            <a
              className="transition-opacity-transform mb-1 inline-block h-4 w-4 translate-x-[-8px] -rotate-45 align-middle opacity-0 duration-500 ease-in-out group-hover:transform-none group-hover:opacity-70"
              href="https://icons8.com/"
              aria-label="Icons8 Homepage"
            >
              <Icons8Icon />
            </a>
          </li>

          <li className="hover group mb-5">
            <a
              className="pr-2 group-hover:underline"
              href="https://fontawesome.com/"
              aria-label="Font Awesome Homepage"
            >
              Font Awesome (Icons)
            </a>

            <a
              className="transition-opacity-transform mb-1 inline-block h-4 w-4 translate-x-[-8px] -rotate-45 align-middle opacity-0 duration-500 ease-in-out group-hover:transform-none group-hover:opacity-70"
              href="https://fontawesome.com/"
              aria-label="Font Awesome Homepage"
            >
              <FontAwesomeIcon />
            </a>
          </li>

          <li className="hover group mb-5">
            <a
              className="pr-2 group-hover:underline"
              href="https://brittanychiang.com/"
              aria-label="Brittany Chiang Website"
            >
              Brittany Chiang (Inspired the programming experience component)
            </a>

            <a
              className="transition-opacity-transform mb-2 inline-block h-4 w-4 translate-x-[-8px] -rotate-45 align-middle opacity-0 duration-500 ease-in-out group-hover:transform-none group-hover:opacity-70"
              href="https://brittanychiang.com/"
              aria-label="Brittany Chiang Website"
            >
              <BrittanyChiangIcon />
            </a>
          </li>

          <li className="hover group mb-5">
            <a
              className="pr-2 group-hover:underline"
              href="https://marguerite.io/"
              aria-label="Marguerite Roth Website"
            >
              Marguerite Roth (Inspired the bookshelf component)
            </a>

            <a
              className="transition-opacity-transform relative mb-1 inline-block h-6 w-6 translate-x-[-8px] -rotate-45 align-middle opacity-0 duration-500 ease-in-out group-hover:transform-none group-hover:opacity-70"
              href="https://marguerite.io/"
              aria-label="Marguerite Roth Website"
            >
              <Image
                src={Marguerite}
                fill
                sizes="24px"
                alt="Marguerite Roth Icon"
              />
            </a>
          </li>

          <li className="hover group mb-5">
            <a
              className="pr-2 group-hover:underline"
              href="https://fresh.deno.dev/"
              aria-label="Fresh Framework Homepage"
            >
              Fresh Framework (Dripping Design + Project Showcase)
            </a>

            <a
              className="transition-opacity-transform mb-1 inline-block h-6 w-6 translate-x-[-8px] -rotate-45 align-middle opacity-0 duration-500 ease-in-out group-hover:transform-none group-hover:opacity-70"
              href="https://fresh.deno.dev/"
              aria-label="Fresh Framework Homepage"
            >
              <FreshIcon />
            </a>
          </li>

          <li className="hover group mb-5">
            <a
              className="pr-2 group-hover:underline"
              href="https://www.joshwcomeau.com/"
              aria-label="Josh Comeau Website"
            >
              Josh Comeau (Moon Icon)
            </a>

            <a
              className="transition-opacity-transform relative mb-1 inline-block h-6 w-6 translate-x-[-8px] -rotate-45 align-middle opacity-0 duration-500 ease-in-out group-hover:transform-none group-hover:opacity-70"
              href="https://www.joshwcomeau.com/"
              aria-label="Josh Comeau Website"
            >
              <Image
                src={JoshComeau}
                fill
                sizes="24px"
                alt="Josh Comeau Icon"
              />
            </a>
          </li>

          <li className="hover group mb-5">
            <a
              className="pr-2 group-hover:underline"
              href="https://nuxt.com/"
              aria-label="Nuxt Framework Homepage"
            >
              Nuxt Framework (Technologies Section)
            </a>

            <a
              className="transition-opacity-transform inline-block h-6 w-6 translate-x-[-8px] -rotate-45 pt-1 align-middle opacity-0 duration-500 ease-in-out group-hover:transform-none group-hover:opacity-70"
              href="https://nuxt.com/"
              aria-label="Nuxt Framework Homepage"
            >
              <NuxtIcon />
            </a>
          </li>
        </ul>
      </div>
    </SectionLayout>
  );
}
