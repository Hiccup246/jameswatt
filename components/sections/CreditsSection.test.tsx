import { render } from "@testing-library/react";
import CreditsSection from "./CreditsSection";

describe("CreditsSection", () => {
  it("renders correctly", () => {
    const { container } = render(<CreditsSection />);

    expect(container).toMatchSnapshot();
  });
});
