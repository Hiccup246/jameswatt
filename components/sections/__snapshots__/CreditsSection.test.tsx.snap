// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreditsSection renders correctly 1`] = `
<div>
  <div
    class="flex w-full flex-row justify-center pt-12 pb-20 pt-20 bg-brown dark:bg-darkgrey text-black dark:text-softwhite"
  >
    <section
      class="mx-auto flex w-9/12 flex-col justify-start"
    >
      <h1
        class="text-center text-2xl font-bold sm:text-3xl"
      >
        The Work Which Inspired This Site
      </h1>
      <h2
        class="mb-16 text-center"
      >
        "Good artists borrow, great artists steal" - Pablo Picasso
      </h2>
      <div
        class="text-center"
      >
        <ul>
          <li
            class="hover group mb-5"
          >
            <a
              aria-label="Icons8 Homepage"
              class="pr-2 group-hover:underline"
              href="https://icons8.com/"
            >
              Icons8 (Icons)
            </a>
            <a
              aria-label="Icons8 Homepage"
              class="transition-opacity-transform mb-1 inline-block h-4 w-4 translate-x-[-8px] -rotate-45 align-middle opacity-0 duration-500 ease-in-out group-hover:transform-none group-hover:opacity-70"
              href="https://icons8.com/"
            >
              <svg
                fill="#1fb141"
                viewBox="0 0 14 14"
              >
                <path
                  d="M7 0H0v14h7V0z"
                />
                <path
                  d="M10.5 7a3.5 3.5 0 1 0 0-7 3.5 3.5 0 0 0 0 7zm0 7a3.5 3.5 0 1 0 0-7 3.5 3.5 0 0 0 0 7z"
                />
              </svg>
            </a>
          </li>
          <li
            class="hover group mb-5"
          >
            <a
              aria-label="Font Awesome Homepage"
              class="pr-2 group-hover:underline"
              href="https://fontawesome.com/"
            >
              Font Awesome (Icons)
            </a>
            <a
              aria-label="Font Awesome Homepage"
              class="transition-opacity-transform mb-1 inline-block h-4 w-4 translate-x-[-8px] -rotate-45 align-middle opacity-0 duration-500 ease-in-out group-hover:transform-none group-hover:opacity-70"
              href="https://fontawesome.com/"
            >
              <svg
                fill="#538dd7"
                viewBox="0 0 448 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M448 48V384c-63.1 22.5-82.3 32-119.5 32c-62.8 0-86.6-32-149.3-32c-20.6 0-36.6 3.6-51.2 8.2v-64c14.6-4.6 30.6-8.2 51.2-8.2c62.7 0 86.5 32 149.3 32c20.4 0 35.6-3 55.5-9.3v-208c-19.9 6.3-35.1 9.3-55.5 9.3c-62.8 0-86.6-32-149.3-32c-50.8 0-74.9 20.6-115.2 28.7V448c0 17.7-14.3 32-32 32s-32-14.3-32-32V64C0 46.3 14.3 32 32 32s32 14.3 32 32V76.7c40.3-8 64.4-28.7 115.2-28.7c62.7 0 86.5 32 149.3 32c37.1 0 56.4-9.5 119.5-32z"
                />
              </svg>
            </a>
          </li>
          <li
            class="hover group mb-5"
          >
            <a
              aria-label="Brittany Chiang Website"
              class="pr-2 group-hover:underline"
              href="https://brittanychiang.com/"
            >
              Brittany Chiang (Inspired the programming experience component)
            </a>
            <a
              aria-label="Brittany Chiang Website"
              class="transition-opacity-transform mb-2 inline-block h-4 w-4 translate-x-[-8px] -rotate-45 align-middle opacity-0 duration-500 ease-in-out group-hover:transform-none group-hover:opacity-70"
              href="https://brittanychiang.com/"
            >
              <svg
                id="logo"
                role="img"
                viewBox="0 0 84 96"
                xmlns="http://www.w3.org/2000/svg"
              >
                <title>
                  Logo
                </title>
                <g
                  fill="#64ffda"
                  transform="translate(-8.000000, -2.000000)"
                >
                  <g
                    transform="translate(11.000000, 5.000000)"
                  >
                    <polygon
                      fill="black"
                      id="Shape"
                      points="39 0 0 22 0 67 39 90 78 68 78 23"
                      stroke="#64ffda"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="5"
                    />
                    <path
                      d="M45.691667,45.15 C48.591667,46.1 50.691667,48.95 50.691667,52.2 C50.691667,57.95 46.691667,61 40.291667,61 L28.541667,61 L28.541667,30.3 L39.291667,30.3 C45.691667,30.3 49.691667,33.15 49.691667,38.65 C49.691667,41.95 47.941667,44.35 45.691667,45.15 Z M33.591667,43.2 L39.241667,43.2 C42.791667,43.2 44.691667,41.85 44.691667,38.95 C44.691667,36.05 42.791667,34.8 39.241667,34.8 L33.591667,34.8 L33.591667,43.2 Z M33.591667,47.5 L33.591667,56.5 L40.191667,56.5 C43.691667,56.5 45.591667,54.75 45.591667,52 C45.591667,49.2 43.691667,47.5 40.191667,47.5 L33.591667,47.5 Z"
                      stroke="#64ffda"
                    />
                  </g>
                </g>
              </svg>
            </a>
          </li>
          <li
            class="hover group mb-5"
          >
            <a
              aria-label="Marguerite Roth Website"
              class="pr-2 group-hover:underline"
              href="https://marguerite.io/"
            >
              Marguerite Roth (Inspired the bookshelf component)
            </a>
            <a
              aria-label="Marguerite Roth Website"
              class="transition-opacity-transform relative mb-1 inline-block h-6 w-6 translate-x-[-8px] -rotate-45 align-middle opacity-0 duration-500 ease-in-out group-hover:transform-none group-hover:opacity-70"
              href="https://marguerite.io/"
            >
              <img
                alt="Marguerite Roth Icon"
                data-nimg="fill"
                decoding="async"
                loading="lazy"
                sizes="24px"
                src="/_next/image?url=%2Fimg.jpg&w=3840&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=16&q=75 16w, /_next/image?url=%2Fimg.jpg&w=32&q=75 32w, /_next/image?url=%2Fimg.jpg&w=48&q=75 48w, /_next/image?url=%2Fimg.jpg&w=64&q=75 64w, /_next/image?url=%2Fimg.jpg&w=96&q=75 96w, /_next/image?url=%2Fimg.jpg&w=128&q=75 128w, /_next/image?url=%2Fimg.jpg&w=256&q=75 256w, /_next/image?url=%2Fimg.jpg&w=384&q=75 384w, /_next/image?url=%2Fimg.jpg&w=640&q=75 640w, /_next/image?url=%2Fimg.jpg&w=750&q=75 750w, /_next/image?url=%2Fimg.jpg&w=828&q=75 828w, /_next/image?url=%2Fimg.jpg&w=1080&q=75 1080w, /_next/image?url=%2Fimg.jpg&w=1200&q=75 1200w, /_next/image?url=%2Fimg.jpg&w=1920&q=75 1920w, /_next/image?url=%2Fimg.jpg&w=2048&q=75 2048w, /_next/image?url=%2Fimg.jpg&w=3840&q=75 3840w"
                style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
              />
            </a>
          </li>
          <li
            class="hover group mb-5"
          >
            <a
              aria-label="Fresh Framework Homepage"
              class="pr-2 group-hover:underline"
              href="https://fresh.deno.dev/"
            >
              Fresh Framework (Dripping Design + Project Showcase)
            </a>
            <a
              aria-label="Fresh Framework Homepage"
              class="transition-opacity-transform mb-1 inline-block h-6 w-6 translate-x-[-8px] -rotate-45 align-middle opacity-0 duration-500 ease-in-out group-hover:transform-none group-hover:opacity-70"
              href="https://fresh.deno.dev/"
            >
              <svg
                viewBox="0 0 40 40"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M34.092 8.845C38.929 20.652 34.092 27 30 30.5c1 3.5-2.986 4.222-4.5 2.5-4.457 1.537-13.512 1.487-20-5C2 24.5 4.73 16.714 14 11.5c8-4.5 16-7 20.092-2.655Z"
                  fill="#FFDB1E"
                  stroke="#ffbf00"
                  stroke-width="2px"
                />
                <path
                  d="M14 11.5c6.848-4.497 15.025-6.38 18.368-3.47C37.5 12.5 21.5 22.612 15.5 25c-6.5 2.587-3 8.5-6.5 8.5-3 0-2.5-4-5.183-7.75C2.232 23.535 6.16 16.648 14 11.5Z"
                  fill="#fff"
                  stroke="#ffbf00"
                  stroke-width="2px"
                />
                <path
                  d="M28.535 8.772c4.645 1.25-.365 5.695-4.303 8.536-3.732 2.692-6.606 4.21-7.923 4.83-.366.173-1.617-2.252-1.617-1 0 .417-.7 2.238-.934 2.326-1.365.512-4.223 1.29-5.835 1.29-3.491 0-1.923-4.754 3.014-9.122.892-.789 1.478-.645 2.283-.645-.537-.773-.534-.917.403-1.546C17.79 10.64 23 8.77 25.212 8.42c.366.014.82.35.82.629.41-.14 2.095-.388 2.503-.278Z"
                  fill="#FFE600"
                />
                <path
                  d="M14.297 16.49c.985-.747 1.644-1.01 2.099-2.526.566.121.841-.08 1.29-.701.324.466 1.657.608 2.453.701-.715.451-1.057.852-1.452 2.106-1.464-.611-3.167-.302-4.39.42Z"
                  fill="#fff"
                />
              </svg>
            </a>
          </li>
          <li
            class="hover group mb-5"
          >
            <a
              aria-label="Josh Comeau Website"
              class="pr-2 group-hover:underline"
              href="https://www.joshwcomeau.com/"
            >
              Josh Comeau (Moon Icon)
            </a>
            <a
              aria-label="Josh Comeau Website"
              class="transition-opacity-transform relative mb-1 inline-block h-6 w-6 translate-x-[-8px] -rotate-45 align-middle opacity-0 duration-500 ease-in-out group-hover:transform-none group-hover:opacity-70"
              href="https://www.joshwcomeau.com/"
            >
              <img
                alt="Josh Comeau Icon"
                data-nimg="fill"
                decoding="async"
                loading="lazy"
                sizes="24px"
                src="/_next/image?url=%2Fimg.jpg&w=3840&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=16&q=75 16w, /_next/image?url=%2Fimg.jpg&w=32&q=75 32w, /_next/image?url=%2Fimg.jpg&w=48&q=75 48w, /_next/image?url=%2Fimg.jpg&w=64&q=75 64w, /_next/image?url=%2Fimg.jpg&w=96&q=75 96w, /_next/image?url=%2Fimg.jpg&w=128&q=75 128w, /_next/image?url=%2Fimg.jpg&w=256&q=75 256w, /_next/image?url=%2Fimg.jpg&w=384&q=75 384w, /_next/image?url=%2Fimg.jpg&w=640&q=75 640w, /_next/image?url=%2Fimg.jpg&w=750&q=75 750w, /_next/image?url=%2Fimg.jpg&w=828&q=75 828w, /_next/image?url=%2Fimg.jpg&w=1080&q=75 1080w, /_next/image?url=%2Fimg.jpg&w=1200&q=75 1200w, /_next/image?url=%2Fimg.jpg&w=1920&q=75 1920w, /_next/image?url=%2Fimg.jpg&w=2048&q=75 2048w, /_next/image?url=%2Fimg.jpg&w=3840&q=75 3840w"
                style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
              />
            </a>
          </li>
          <li
            class="hover group mb-5"
          >
            <a
              aria-label="Nuxt Framework Homepage"
              class="pr-2 group-hover:underline"
              href="https://nuxt.com/"
            >
              Nuxt Framework (Technologies Section)
            </a>
            <a
              aria-label="Nuxt Framework Homepage"
              class="transition-opacity-transform inline-block h-6 w-6 translate-x-[-8px] -rotate-45 pt-1 align-middle opacity-0 duration-500 ease-in-out group-hover:transform-none group-hover:opacity-70"
              href="https://nuxt.com/"
            >
              <svg
                data-v-61b8e4cb=""
                viewBox="0 0 150 90"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M74.2284 88.3575H123.414C124.977 88.3577 126.511 87.9588 127.864 87.2011C129.217 86.4433 130.341 85.3534 131.121 84.0409C131.902 82.7284 132.313 81.2396 132.312 79.7242C132.312 78.2089 131.899 76.7204 131.118 75.4086L98.0856 19.912C97.3048 18.5998 96.1817 17.5101 94.829 16.7525C93.4764 15.9948 91.942 15.596 90.38 15.596C88.8181 15.596 87.2836 15.9948 85.931 16.7525C84.5784 17.5101 83.4552 18.5998 82.6745 19.912L74.2284 34.1116L57.7148 6.34469C56.9334 5.03259 55.8097 3.94306 54.4567 3.18557C53.1037 2.42808 51.569 2.0293 50.0068 2.0293C48.4446 2.0293 46.9099 2.42808 45.5569 3.18557C44.2039 3.94306 43.0803 5.03259 42.2989 6.34469L1.1947 75.4086C0.41273 76.7204 0.000680577 78.2089 8.42355e-07 79.7242C-0.000678893 81.2396 0.410035 82.7284 1.19083 84.0409C1.97163 85.3534 3.09498 86.4433 4.44789 87.2011C5.80081 87.9588 7.33558 88.3577 8.89785 88.3575H39.7727C52.0057 88.3575 61.0271 83.1454 67.2346 72.9769L82.3054 47.6603L90.3776 34.1116L114.604 74.8088H82.3054L74.2284 88.3575ZM39.2694 74.7948L17.7227 74.7902L50.0212 20.5304L66.1369 47.6603L55.3468 65.7934C51.2244 72.391 46.5411 74.7948 39.2694 74.7948Z"
                  fill="#00DC82"
                />
              </svg>
            </a>
          </li>
        </ul>
      </div>
    </section>
  </div>
</div>
`;
